src/ctap/ctap1.rs

```diff
@@ -118,11 +119,16 @@ impl TryFrom<&[u8]> for U2fCommand {
     type Error = Ctap1StatusCode;
 
     fn try_from(message: &[u8]) -> Result<Self, Ctap1StatusCode> {
-        if message.len() < Ctap1Command::APDU_HEADER_LEN as usize {
-            return Err(Ctap1StatusCode::SW_WRONG_DATA);
-        }
-
-        let (apdu, payload) = message.split_at(Ctap1Command::APDU_HEADER_LEN as usize);
+        let apdu: APDU = match APDU::try_from(message) {
+            Ok(apdu) => apdu,
+            // Todo: Better conversion between ApduStatusCode and Ctap1StatusCode
+            // Maybe use TryFrom?
```

Comment:
Our `TODO` style would be
> // TODO(kaczmarczyck)

or 
> // TODO(#123456)

for a GitHub issue number. In this case, why not resolve it? By merging these two status codes? I don't think we need both.